build.xml

```diff
@@ -645,6 +645,7 @@ message.
 			encoding="UTF-8"
 			header="&lt;b&gt;Liferay ${lp.version} ${module.dir}&lt;/b&gt;"
 			maxmemory="2048m"
+			noindex="yes"
```

Comment:
@davisaints, please double check if it's still needed when you set the Java API to version 17.